pom.xml

```diff
@@ -233,6 +233,9 @@
                         <exclude>**/uk.gov.companieshouse.account.validator.Runner.java</exclude>
                     </excludes>
                     <skipTests>${skip.unit.tests}</skipTests>
+                    <environmentVariables>
+                        <IXBRL_TO_PDF_URI>http://random.api.url</IXBRL_TO_PDF_URI>
```

Comment:
Is this going to be changed to the correct url?

Reply:
no, it's only for testing purposes